pkg/util/image_utils.go

```diff
@@ -61,6 +73,15 @@ func SortMap(m map[string]struct{}) string {
 	return strings.Join(pairs, " ")
 }
 
+// GetFileSystemForLayer unpacks a layer to local disk
+func GetFileSystemForLayer(layer v1.Layer, root string, whitelist []string) error {
+	contents, err := layer.Uncompressed()
+	if err != nil {
+		return err
+	}
+	return unpackTar(tar.NewReader(contents), root, whitelist)
```

Comment:
As part of switching to go-containerregistry, I removed all the whiteout handling in `unpackTar` (since it gets handled implicitly by `mutate.Extract`). I don't think it should cause any problems here since a tombstoned file would be treated like any normal file would, but just making a note of that here in case you can think of any cases where it would be an issue.